pyads/symbol.py

```diff
@@ -288,3 +270,44 @@ def get_type_from_str(type_str: str) -> Any:
         # error when they are being addressed
 
         return None
+
+    @property
+    def auto_update(self) -> Any:
+        """Return True if auto_update is enabled for this symbol."""
+        return self._auto_update_handle is not None
+
+    @auto_update.setter
+    def auto_update(self, value: bool) -> None:
+        """Enable or disable auto-update of the buffered value.
+
+        This automatic update is done through a device notification. This
+        can be efficient when a remote variables changes its values less often
+        than your code run.
+
+        Clearing all device notifications will also disable auto-update.
+        Automatic update is disabled by default.
+        """
+        if value and self._auto_update_handle is None:
+            self._auto_update_handle = self.add_device_notification(
+                self._value_callback
+            )
+        elif not value and self._auto_update_handle is not None:
+            self.del_device_notification(self._auto_update_handle)
+            self._auto_update_handle = None
+
+    @property
+    def value(self) -> Any:
+        """Return the current value of the symbol."""
+        return self._value
+
+    @value.setter
+    def value(self, val: Any) -> None:
+        """Set the current value of the symbol.
+
+        If auto_update is True then the the write command will be called automatically.
+
+        """
+        self._value = val
+        # write value to plc if auto_update is enabled
+        if self._auto_update_handle is not None:
```

Comment:
I think `if self.auto_update` it's a little easier to read.

Reply:
That's a good suggestion. I implemented the auto_update property after the value property so I didn' t see this. I will change it.